### ISSUE TITLE
Breakpoint error when launch Steam on Windows 7

### ISSUE THREAD

**Issue by Dachoni03** [vxiiduu/VxKex#235]
In this screenshot/picture you can see the error.

![Steam_breakpoint_error](https://github.com/vxiiduu/VxKex/assets/106074709/457d9057-1e79-425a-b8ca-691310988f39)

**Comment by Dachoni03** [comment]
This only happens when I successfully logged into my account.

**Comment by CommonLoon102** [comment]
Isn't Steam working on Win7 without VxKex?

**Comment by i486** [comment]
Yeap. I've just updated my Steam client to the one released yesterday and it appears to be working fine.

**Comment by CommonLoon102** [comment]
Ah OK, so when Steam won't work anymore on Win7, VxKex won't solve the issue. Because it doesn't even work now.

**Comment by Dachoni03** [comment]
Yeah sad

**Comment by i486** [comment]
I did a quick test on Steam using VxKex out of curiosity. 

Only checking the "Enable VxKex for this program" option. Steam itself seems to function normally, even when logging into different accounts, which wasn't the case for OP. However, all games crash upon launch through Steam.

![image](https://github.com/vxiiduu/VxKex/assets/625325/f299ce91-2637-499b-802e-763939d99399)

Things got a bit strange after enabling "Disable VxKex for child processes". Most games can now be launched successfully, but some still crash upon launching:

CS 1.6 Beta crashes.
L4D2 crashes.
PAYDAY 2 crashes.

[Logs for only Enable VxKex for this program.zip](https://github.com/vxiiduu/VxKex/files/15258053/Logs.for.only.Enable.VxKex.for.this.program.zip)

[Log for Enable VxKex for this program and NoChildProcess.zip](https://github.com/vxiiduu/VxKex/files/15258052/Log.for.Enable.VxKex.for.this.program.and.NoChildProcess.zip)

**Comment by metapea** [comment]
For me, i got the same errors when using steam normally (aka without VxKex) in VxKex

**Comment by i486** [comment]
> For me, i got the same errors when using steam normally (aka **without VxKex**) in **VxKex**

Typo?

**Comment by metapea** [comment]
> > For me, i got the same errors when using steam normally (aka **without VxKex**) in **VxKex**
> 
> Typo?

no

**Comment by CommonLoon102** [comment]
> > > For me, i got the same errors when using steam normally (aka **without VxKex**) in **VxKex**
> > 
> > 
> > Typo?
> 
> no

syntax error then

**Comment by metapea** [comment]
> > > > For me, i got the same errors when using steam normally (aka **without VxKex**) in **VxKex**
> > > 
> > > 
> > > Typo?
> > 
> > 
> > no
> 
> syntax error then

Testing this again with a more recent version, i know that isn't the case since even though it's the same as before, the tray for the notification area works now when using steam with VxKex.
Also when i use VxKex with windows 10 reporting i get this after stream auto restart on it's own multiple time:
![7](https://github.com/vxiiduu/VxKex/assets/35351174/a7c79ce3-ebe2-4ca8-b164-dbb618da9942)

**Comment by Dachoni03** [comment]
> > > > > For me, i got the same errors when using steam normally (aka **without VxKex**) in **VxKex**
> > > > 
> > > > 
> > > > Typo?
> > > 
> > > 
> > > no
> > 
> > 
> > syntax error then
> 
> Testing this again with a more recent version, i know that isn't the case since even though it's the same as before, the tray for the notification area works now when using steam with VxKex. Also when i use VxKex with windows 10 reporting i get this after stream auto restart on it's own multiple times: ![7](https://private-user-images.githubusercontent.com/35351174/335308653-a7c79ce3-ebe2-4ca8-b164-dbb618da9942.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0MDc2ODAsIm5iZiI6MTcxOTQwNzM4MCwicGF0aCI6Ii8zNTM1MTE3NC8zMzUzMDg2NTMtYTdjNzljZTMtZWJlMi00Y2E4LWIxNjQtZGJiNjE4ZGE5OTQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI2VDEzMDk0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJhYjA1MWY1MzhlOTA0MDhiMmExNjc3ZDZjY2RjN2IyMDgzNzBkNWFmNGI4ODFkZjE4MjdiNDY4ZjhmYWRkZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dM1iQgx1Fx_6iOxTK3J2Xu9G4YONF_-wI4dWnJzDwJI)

Yeah, sad I will try to do something, but I don't promise, but I do not promise . :)
